assets/js/admin.js

```diff
@@ -94,7 +94,7 @@ jQuery(document).ready(function($) {
     // Set the tab showing on the form and persist the tab
     $( '.js-a0-settings-tabs' ).click( function () {
         window.location.hash = '';
-        var tabHref = $( this ).attr( 'aria-controls' );
+        var tabHref = $( this ).attr( 'aria-controls' ).trim();
```

Comment:
Found an issue with whitespace formatting 😮 corrected the issue but this will prevent it from happening again.